CMakeLists.txt

```diff
@@ -39,7 +39,13 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 if (MSVC)
 	set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT CemuBin)
 	# floating point model: precise, fiber safe optimizations
-	add_compile_options(/EHsc /fp:precise /GT)
+	add_compile_options(/EHsc /fp:precise)
+	if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+		# Speeds up static linking (especially helpful in incremental compilation)
+		set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY STATIC_LIBRARY_OPTIONS /llvmlibthin)
```

Comment:
I did a quick search and it seems that `/llvmlibthin` is only supported when using the LLD linker with the link.exe on Windows. Does using clang-cl necessarily mean that one is also using LLD? Is it possible to use clang-cl with MSVC's link.exe?

Reply:
good catch!
it's fixed now.